### PR TITLE
Gov vote flow

### PR DESCRIPTION
**Pull request by Relequestual**
**GitHub Issue:** NA
Related to https://github.com/json-schema-org/community/issues/423

**Summary**: Keeping the governance process loaded in memory might be tricky as we get used to using it. This PR adds a Finite State Machine for the voting process. A visual and executable version of the voting process should help in its understanding.

**Do you think resolving this issue might require an [Architectural Decision Record (ADR)](https://github.com/json-schema-org/community/blob/main/CONTRIBUTING.md#key-architectural-decisions)? (significant or noteworthy)**
No

To preview the [Mermaid diagram](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams) in this PR, click the dots (...) in the top right of the file, then "View File".

### CODE REVIEW

File: docs/governance/supporting-info.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```mermaid
+%% Generated with Stately Studio
+stateDiagram-v2
+  state "votingMachine" as votingMachine {
+    [*] --> votingMachine.Idle
+    votingMachine.Idle --> votingMachine.Awaiting_public_vote_setup : Call TSC Vote Public \ndo / Call for TSC vote
+    votingMachine.Idle --> votingMachine.Awaiting_approval_for_private_vote : Request private vote \ndo / Contact TSC Chairs
+    votingMachine.Awaiting_approval_for_private_vote --> votingMachine.Awaiting_additional_comment_for_counting_votes : Create Issue in private repo\nif [TSC Chair approved]
+    votingMachine.Voting_open --> votingMachine.Awaiting_extension_approval : request_extension \ndo / requestExtension
+    votingMachine.Voting_open --> votingMachine.VotingClosed : xstate.after(604800000)#votingMachine.Voting open \ndo / calculateQuorum
```
Comment (gregsdennis) [inline docs/governance/supporting-info.md:31]: This is rendering weird:

image

**Review by gregsdennis — APPROVED** [review]
This seems like a lot for "how does voting work?"

**Comment by gregsdennis** [comment]
> To preview the [Mermaid diagram](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams) in this PR, click the dots (...) in the top right of the file, then "View File".

You can also just look at the markdown view in the file diff.

File: docs/governance/supporting-info.md (hunk 1 above)
Comment (Relequestual) [reply docs/governance/supporting-info.md:31]: Yeah, unfortunatly it doesn't seem like I can do much about that.
I think what's more important is the flow and what happens in what order.

**Review by benjagm — APPROVED** [review]
Thanks a lot for this Ben. It will help a lot to folks like me.

**Pull request merged**